app/javascript/packs/gist.js

```diff
@@ -0,0 +1,42 @@
+export function embedGists() {
```

Comment:
Refactor (blocking): Pack files are typically a webpack entry point. As well these functions already exist app/javascript/utilities/gist.js in your PR. Also, from the looks of it, nothing is ever called in this file, so it can be removed.

Reply:
hi, thanks for the review, the utility can be removed, but not the pack.